packages/google/src/google-generative-ai-prompt.ts

```diff
@@ -38,3 +38,21 @@ export type GoogleGenerativeAIContentPart =
         fileUri: string;
       };
     };
+
+export interface GoogleGenerativeAIGroundingMetadata {
+  webSearchQueries?: string[] | null;
```

Comment:
I followed the pattern of `nullish` everywhere instead of `optional` in the schema change, after https://github.com/vercel/ai/pull/4038. It makes this interface definition more unsightly as we have to allow `null` everywhere. I didn't actually validate whether `optional` would suffice for some of these. I can do that if you think warranted.